### PR TITLE
TECH/907

### PR DESCRIPTION
**Pull request by xCodeMuse**
PENDING
- [ ] (https://github.com/godspeedsystems/gs-node-service/issues/907)

### CODE REVIEW

File: .template/dot-configs/.env
Comment (mastersilv3r) [inline .template/dot-configs/.env:1]: Please remove admin and mindgrep from here. use <user>:<password>

File: .template/dot-configs/.eslintrc.json [hunk 1]
```diff
@@ -1,21 +1,120 @@
...
+    "jest"
+  ],
   "env": {
     "browser": false,
     "es6": true,
     "node": true
   },
+  "rules": {
+    "as-needed": "off",
+    "no-unused-vars": "off",
```
Comment (mastersilv3r) [inline .template/dot-configs/.eslintrc.json]: on

File: .template/dot-configs/.eslintrc.json [hunk 2]
```diff
@@ -1,21 +1,120 @@
...
+    "@typescript-eslint/no-unused-vars": 0,
```
Comment (mastersilv3r) [inline .template/dot-configs/.eslintrc.json]: on

File: .template/package.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
   "scripts": {
     "dev": "nodemon --watch src -e ts,js,yaml,json -r dotenv ./src/index.ts",
     "gen-crud-api": "npx @godspeedsystems/api-generator",
-    "build": "NODE_ENV=production swc src -d dist && rsync -a --exclude '*.ts' ./src/ ./dist/",
+    "build": "swc src -d dist && rsync -a --exclude '*.ts' ./src/ ./dist/ && rsync -a ./config/ ./dist/config && rsync -a ./src/mappings/ ./dist/mappings && rsync -a ./src/plugins/ ./dist/plugins && node ./node_modules/@godspeedsystems/core/dist/core/transpiler.js",
```
Comment (mastersilv3r) [inline .template/package.json]: Remove transpiler command for now.

**Review by mastersilv3r — CHANGES_REQUESTED** [review]

**Review by kushal-godspeed — APPROVED** [review]

**Pull request merged**